app/models/prosecution_case.rb

```diff
@@ -36,35 +37,13 @@ def to_builder
       prosecution_case.statementOfFactsWelsh statementOfFactsWelsh
       prosecution_case.breachProceedingsPending breachProceedingsPending
       prosecution_case.appealProceedingsPending appealProceedingsPending
-      prosecution_case.defendants Jbuilder.new.array! defendants_builder
-      prosecution_case.caseMarkers Jbuilder.new.array! markers_builder if markers.present?
+      prosecution_case.defendants array_builder(defendants)
+      prosecution_case.caseMarkers array_builder(markers)
       if split_prosecutor_case_references.present?
-        prosecution_case.splitProsecutorCaseReferences Jbuilder.new.array! split_prosecutor_case_references_builder
+        prosecution_case.splitProsecutorCaseReferences split_prosecutor_case_references.map(&:split)
```

Comment:
I've seen `.map(&:method)` in a few places - is this something else we could put in a cocern?